package.json

```diff
@@ -62,15 +62,18 @@
           "scope": "window"
         },
         "xml.catalogs": {
-          "type":"array",
+          "type": "array",
           "default": [],
+          "items": {
```

Comment:
"default": {
                "systemId": "path/to/file.xsd",
                "pattern": "**/*data.xml"
            },

Reply:
Why? Personally I would find that annoying as a user if I selected the autocompletion and it would prepopulate with an object with dummy values that have to manually delete and replace. It's much faster to use autcompletion to set the values, because it will generate empty strings and place your cursor into them.

Also, wouldn't `default` cause the vscode extension API to use that as a default value if it's not set?

Reply:
It's not the default value of xml.fileAssociations, I simply reproduced vscode's json.schemas [default item](https://github.com/Microsoft/vscode/blob/4419709b9bec8ad7cd1d385e6ec4dd7a13ab4bca/extensions/json-language-features/package.json#L39-L44).

Anyways,
```json
"default": {
            "systemId": "",
            "pattern": ""
        },
```
is a much better default than the current one.

Reply:
I don't understand the purpose.

As a user, this is how I add a config value:

![2018-09-14 22 06 42](https://user-images.githubusercontent.com/10532611/45572765-94f61700-b86a-11e8-9315-ff7cdf4ff9cb.gif)

This is what the `default` allows:

![2018-09-14 22 07 44](https://user-images.githubusercontent.com/10532611/45572806-b48d3f80-b86a-11e8-9e6f-46e0a57774da.gif)

What is the point? What meaning has this "default value" as the hint says?
Using this method is a worse experience - I don't see the documentation for the `systemId` and `pattern` properties while editing.
I don't think it would be a better UX to not add this flag, as then users are always guided into the first flow.

Reply:
I liked having to the full skeleton on the 1st complete, instead of doing 3. But ok, I reckon the documentation part of the default value sucks. 
So let's do it your way for now, and see how users respond over time.